packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -178,6 +179,7 @@ const kernelManager = new KernelSessionManager();
 
 let editor: VAceEditorInstance['_editor'] | null;
 const codeText = ref('');
+const llmQuery = ref('');
```

Comment:
Is it worth throwing `codeText`, `llmQuery`, and `llmThoughts` into one object? 
May be cleaner to read as theyre all individual parts of the same `StratifyCode` type.

If you dont think it will be easier to follow, or it adds complexity in a way I am not seeing no pressure

Reply:
Yeah makes sense. I'll look in to it and address them on my follow up PR if that's better.